model-server/src/main/kotlin/org/modelix/model/server/CmdLineArgs.kt

```diff
@@ -33,6 +33,14 @@ internal class CmdLineArgs {
     @Parameter(names = ["-port", "--port"], description = "Set port", converter = IntegerConverter::class)
     var port: Int? = null
 
+    // 10 seconds was the default value in NettyApplicationEngine when this option was implemented.
```

Comment:
Is 10 seconds reasonable for what we have learned from the streaming problems?

Reply:
I think two minutes are more reasonable and should suffice for most projects.
Anothing above that are specially cases.

FYI For the default timout in the client I created: https://issues.modelix.org/issue/MODELIX-855/Increase-default-timeout-for-request-that-pull-version-deltas